### PR TITLE
chore: version bump

### PR DESCRIPTION
**Pull request by BruceClark406**
version bump and export fix

### CODE REVIEW

File: lib/src/shared_model/shared_model.dart [hunk 1]
```diff
@@ -23,6 +23,7 @@ export 'catalog_object_type.dart';
 export 'catalog_query.dart';
 export 'catalog_query_exact.dart';
 export 'catalog_query_items_for_item_option_values_query.dart';
+export 'catalog_query_items_for_item_option_values_query.dart';
```
Comment (mtwichel) [inline lib/src/shared_model/shared_model.dart:26]: Looks like a little oops

**Review by mtwichel — COMMENTED** [review]
Good to merge after the little oops

File: lib/src/shared_model/shared_model.dart (hunk 1 above)
Comment (BruceClark406) [reply lib/src/shared_model/shared_model.dart]: hmmm, strange.
The linter was telling my that is wasn't exported in shared_model.dart
So, I exported it and low and behold there are two.
However, when I remove one of them, the linter freaks out again.
Turns out if a change the order of parameter CatalogQuery the linter stops screaming at me... 
STRANGE!!!

Comment (BruceClark406) [reply lib/src/shared_model/shared_model.dart]: Should be resolved now.

**Review by mtwichel — APPROVED** [review]

**Pull request merged**